firmware/shared/controls/motor_torque_test.cc

```diff
@@ -0,0 +1,80 @@
+#include "motor_torque.h"
+
+#include <iostream>
+
+#include "testing.h"
+
+int main() {
+    using namespace ctrl;
+
+    // Test cases for CalculateMotorTorque
+    // Test Case 1: Smaller Sample Sizes
+    {
+        double raw_torque_values[] = {5, 10, 15, 20};
+        auto result = CalculateMotorTorque(raw_torque_values, 4, 1.5, 1.2);
```

Comment:
Use "structures bindings" to unpack the tuple into separate variables here rather than with the get method 

https://en.cppreference.com/w/cpp/language/structured_binding